### ISSUE TITLE
Help needed - after PH Git-f725812de needed update source for FirmwarePlugin and LiveDumpPlugin

### ISSUE THREAD

**Issue by VictorVG** [processhacker/plugins-extra#74]
After PH Git [**f725812de**](https://github.com/processhacker/processhacker/commit/f725812de0519c67cae01d11dca940b6053b654b) needed update source for FirmwarePlugin and  LiveDumpPlugin. But I try fix it:

For FirmwarePlugin I replace in to files:

dialog.c strings 271 and 272:

```
           SendMessage(hwndDlg, WM_SETICON, ICON_SMALL, (LPARAM)PH_LOAD_SHARED_ICON_SMALL(PhInstanceHandle, MAKEINTRESOURCE(PHAPP_IDI_PROCESSHACKER)));
           SendMessage(hwndDlg, WM_SETICON, ICON_BIG, (LPARAM)PH_LOAD_SHARED_ICON_LARGE(PhInstanceHandle, MAKEINTRESOURCE(PHAPP_IDI_PROCESSHACKER)));
```

up to string 271:

`            PhSetApplicationWindowIcon(hwndDlg);`

and editor.c strings 134 and 135

```
            SendMessage(hwndDlg, WM_SETICON, ICON_SMALL, (LPARAM)PH_LOAD_SHARED_ICON_SMALL(PhInstanceHandle, MAKEINTRESOURCE(PHAPP_IDI_PROCESSHACKER)));
            SendMessage(hwndDlg, WM_SETICON, ICON_BIG, (LPARAM)PH_LOAD_SHARED_ICON_LARGE(PhInstanceHandle, MAKEINTRESOURCE(PHAPP_IDI_PROCESSHACKER)));
```

up to string 134:

`            PhSetApplicationWindowIcon(hwndDlg);`

But for LiveDumpPlugin

dialog.c string 142:

`    config.hMainIcon = PH_LOAD_SHARED_ICON_LARGE(PhInstanceHandle, MAKEINTRESOURCE(PHAPP_IDI_PROCESSHACKER));`

I down know that's fix build and needed help as fix it.

Best regards, Victor.

**Comment by dmex** [comment]
> LiveDumpPlugin

This plugin was merged into the main program and should be removed.

**Comment by VictorVG** [comment]
Ok! Big thanks! Well, it's easier for me - I will remove it from the toolkit, and we will be considered a little faster :)

**P.S.**

I mean that the other day I upgraded all the machines on the LAN, and put the Xeon E5-2667 / C602J / 16 Gb REG ECC DDR3-1866 / GTX 1060 on the work one and everything started to work right away. The main working program - CAD algorithm which is optimized for 10 - 13 threads gives a peak in terms of speed just on a CPU like mine. Guys guessed right with a gift. :) I only waited two weeks for him to pay, and then, for a week she lay motionless in the mail, but in general it turned out not bad - a workstation with characteristics at the level of DELL Precession T5600 was only assembled by my own hands, and understandably at times cheaper, and most importantly for my tasks.:)

**Comment by VictorVG** [comment]
Ok! Big thanks! Well, it's easier for me - I will remove it from the toolkit, and we will be considered a little faster :)

**P.S.**

I mean that the other day I upgraded all the machines on the LAN, and put the Xeon E5-2667 / C602J / 16 Gb REG ECC DDR3-1866 / GTX 1060 on the work one and everything started to work right away. The main working program - CAD algorithm which is optimized for 10 - 13 threads gives a peak in terms of speed just on a CPU like mine. Guys guessed right with a gift. :) I only waited two weeks for him to pay, and then, for a week she lay motionless in the mail, but in general it turned out not bad - a workstation with characteristics at the level of DELL Precession T5600 was only assembled by my own hands, and understandably at times cheaper, and most importantly for my tasks.:)

**Comment by VictorVG** [comment]
Ok! Big thanks! Well, it's easier for me - I will remove it from the toolkit, and we will be considered a little faster :)

**P.S.**

I mean that the other day I upgraded all the machines on the LAN, and put the Xeon E5-2667 / C602J / 16 Gb REG ECC DDR3-1866 / GTX 1060 on the work one and everything started to work right away. The main working program - CAD algorithm which is optimized for 10 - 13 threads gives a peak in terms of speed just on a CPU like mine. Guys guessed right with a gift. :) I only waited two weeks for him to pay, and then, for a week she lay motionless in the mail, but in general it turned out not bad - a workstation with characteristics at the level of DELL Precession T5600 was only assembled by my own hands, and understandably at times cheaper, and most importantly for my tasks.:)
